word_guess.rb

```diff
@@ -0,0 +1,192 @@
+             ~      ~~~--__            "+"**       *****".colorize(:red)+"       ****".colorize(:light_red)+"     o  "+"*".colorize(:red)+"
+                   ______  (@|   "+"*****".colorize(:light_yellow)+"**  ****".colorize(:light_red)+"         ***".colorize(:red)+"   _  _   º "+"**".colorize(:red)+"
+                  /"+"******".colorize(:light_yellow)+"~~~~||"+"*****".colorize(:light_yellow)+"******".colorize(:light_red)+"*********  ******".colorize(:red)+" (.|( ) ✨
+          |       `--____"+"**********".colorize(:light_yellow)+"*********".colorize(:light_red)+"******** ******".colorize(:red)+"  (_Y|_/ "+"** **".colorize(:red)+"
+         / ~~~--_____    ~~~/ "+"******".colorize(:light_yellow)+"**********".colorize(:light_red)+"*********** **".colorize(:red)+"  /_| "+"**      **".colorize(:red)+"
+                     `~~~~~         "+"***".colorize(:light_yellow)+"*********".colorize(:light_red)+"*********".colorize(:red)+"       L    "+" ***".colorize(:red)+"
+                                          **".colorize(:light_yellow)+"**    *****".colorize(:light_red)+"  ***** ****".colorize(:red)+"
+                                            "+"***".colorize(:red)+"
+
+    "
+    elsif @tally == 4
+    puts "
+    <~>                          ___________
+     | |,_____                  (BWAHAHAAHA!)
+           ___`|                 ⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻⁻
+           |"+"('>".colorize(:red)+"|`-__            o
+             ~      ~~~--__   º        "+"**              ***".colorize(:light_red).blink+"
+                   ______  (@|   "+"*".colorize(:light_yellow).blink+"******  *".colorize(:light_red).blink+"***    *******    ******".colorize(:red).blink+"
+                  /"+"******".colorize(:light_yellow).blink+"~~~~||"+"*".colorize(:light_yellow).blink+"***********".colorize(:light_red).blink+"**********************".colorize(:red).blink+"
+          |       `--____"+"*******".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"*****************************".colorize(:red).blink+"
+         / ~~~--_____    ~~~/ "+"***".colorize(:light_yellow).blink+"******".colorize(:light_red).blink+"******************************".colorize(:red).blink+"
+                     `~~~~~         "+"*".colorize(:light_yellow).blink+"*********".colorize(:light_red).blink+"********************".colorize(:red).blink+"
+                                          "+"****    *********".colorize(:light_red).blink+"*****".colorize(:red).blink+"
+                                            "+"***       ********".colorize(:light_red).blink+"***".colorize(:red).blink+"
+                                                            "+"****".colorize(:light_red).blink+"****".colorize(:red).blink+"
+
+    ".colorize(:red).blink
+    puts "The word was: #{@current_word}".colorize(:magenta)
+    end
+  end
+
+def check_letter
+  #we ask for user input using gets.chomp
+  @tally = 0
```

Comment:
I'm curious why you set this variable to 0 again here since it is already set to 0 in the `initialize`